web/src/Web.App/Views/Error/Forbidden.cshtml

```diff
@@ -27,7 +27,7 @@
         </p>
 
         <p class="govuk-body">
-            If you are still having difficulty accessing this page, email us at BetterFinancialReporting.COMMS@education.gov.uk with full details of the issue you are experiencing.
+            If you are still having difficulty accessing this page, email us at @Constants.SupportEmail with full details of the issue you are experiencing.
```

Comment:
Do we need `mailto:`s here and below?

Reply:
No. It was discussed in refinement, and we shouldn't be using mailto.